x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -147,9 +146,25 @@ for the file realm and the <<security-api-put-user,create users>> and
 <<security-api-change-password,change password>> APIs for the native and reserved realms.
 Other types of realms are not affected and do not require any changes.
 
+[discrete]
+==== Cached Password Hashing
```

Comment:
I think we need add a dedicated anchor for this section in addition to the new separate title to be more resistant to potential future changes to page structure or wording. 
```suggestion
[[fips-cached-password-hashing]]
==== Cached Password Hashing
```

Reply:
will add `[[fips-stored-password-hashing]]` to the other section as well